aiida_nanotech_empa/utils/pymol_render.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+
+import ase.io
+import ase.io.cube
+
+import os
+import tempfile
+
+from PIL import Image, ImageOps
+
+from pymol import cmd  # pylint: disable=import-error
+
+
+def load_cube_atoms(cube_file):
```

Comment:
can we reuse the functions from `pp_cube_utils` here?

Reply:
At the moment, I don't see a way. Pymol has it's own internal method to load the volumetric data from the cube file, and i just need to explicitly read the atomic positions from the cube (`pp_cube_utils` doesn't provide this). Although, we could move `load_cube_atoms` to `pp_cube_utils` and make it a general cube utilities module (and call it `cube_utils`).

Reply:
> Although, we could move `load_cube_atoms` to `pp_cube_utils` and make it a general cube utilities module (and call it `cube_utils`).

I would at least do this and see further how these guys can be integrated.

Reply:
agree, I will implement